tests/FileTest.php

```diff
@@ -7,12 +7,17 @@
 use RuntimeException;
 use Yiisoft\Mailer\File;
 
-class FileTest extends TestCase
+use function strlen;
+use function strpos;
+
+final class FileTest extends TestCase
 {
     public function testFromContent(): void
     {
         $file1 = File::fromContent('');
 
+        $this->assertSame(36, strlen($file1->id()));
```

Comment:
May be check id via regular expression?
Only suggestion :)

Reply:
This case is specially designed for MSI, it checks exactly the hash length.